proposals/lambda-method-group-defaults.md

```diff
@@ -1,15 +1,19 @@
-# Default Parameters for Lambdas and Method Groups
+# Default and Variable Parameters for Lambdas and Method Groups
 
 ## Summary
 
 [summary]: #summary
 
-To build on top of the lambda improvements introduced in C#10 (see [relevant background](#relevant-background)), we propose adding support for default parameter values in lambdas. This would enable users to implement the following lambdas:
+To build on top of the lambda improvements introduced in C# 10 (see [relevant background](#relevant-background)), we propose adding support for default parameter values and [`params` arrays](https://docs.microsoft.com/dotnet/csharp/language-reference/keywords/params) in lambdas. This would enable users to implement the following lambdas:
```

Comment:
I don't think we really need the docs link. If we want to link to anything, we should link to the csharpstandard repo, but I don't think a link is necessary at all here.

Reply:
OK, thanks. Removed the link: b811c74f82222febb9630d215ef68698edea21d2